test/screenshotter/test.html

```diff
@@ -8,6 +8,9 @@
       #math, #pre, #post {
         font-size: 4em;
       }
+      body {
+        font-family: "DejaVu Serif",serif;
```

Comment:
Looks like: with this setting (or even just the first half of it), the Firefox screenshots look no different than without, but the Chrome screenshot switch from some sans-serif default to something that looks just like it does in Firefox.